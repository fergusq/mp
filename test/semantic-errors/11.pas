program p;
begin while 2 do writeln("");
end.
