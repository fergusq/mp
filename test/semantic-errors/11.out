Semantic error: Type mismatch: while condition must be boolean: expected boolean, got integer
